PicaSWB.js

```diff
@@ -39,6 +48,16 @@ var nachnameMapping = {
 };
 var nameMapping = {
```

Comment:
Okay, wenn ich Dich richtig verstehe dann wird da wird ein zusätzliches Feld mit dem Link ausgegeben, welches die Suchabfrage enthält. Duch ein Klicken auf diesem Link (in einem entsprechenden Editor) kann der Bearbeiter leicht die Nummer hinzufügen.